docs/modules/sql/pages/select-statement.adoc

```diff
@@ -21,8 +21,9 @@ Any row that doesn't satisfy the condition is eliminated from the result.
 
 == Sorting
 
-You can use standard the SQL clauses `ORDER BY`, `LIMIT`, and `OFFSET` to sort and limit the result set.
-Note that, for this, you need to add indexes to the map objects to be queried and sorted.
+You can use the standard SQL clauses `ORDER BY`, `LIMIT`, and `OFFSET` to sort and limit the result set.
+
+WARNING: Note that, you must add indexes to the map objects to be queried and sorted.
```

Comment:
I think we could be even clearer with an example:


WARNING: Note that, you must add sorted indexes to the map object's fields sorted by. For example, suppose you have a query like `SELECT * FROM persons ORDER BY name ASC`, there's has to be a sorted index on the `name` field like:
```
hazelcast:
  map:
    default:
      indexes:
        - type: SORTED
            attributes:
              - "name"
```

Please, verify with @petrpleshachkov that it has to be a sorted index, not just hash. But in general, I would add a specific example as it's much easier to understand.

Reply:
Preferably, the configuration should be present in XML, YAML and programmatic configuration.